### ISSUE TITLE
Venus 3

### ISSUE THREAD

**Issue by buddhafragt** [stormmurdoc/victron_sdm630_bridge#10]
Someone test it with Venus 3.x?
Is it possible to use it for more than 1 SDM630?
What is with the SDM72?

**Comment by stormmurdoc** [comment]
Yes - 3.4.1 works fine. Can not confirm that this will work on your SDM72 'cause I have no one.
